registration/approval.py

```diff
@@ -56,6 +57,8 @@ def accept_application(application, appserver):
 
     send_welcome_email(application)
 
+    betapplication_accepted.send(sender=None, application=application)
+
     application.status = BetaTestApplication.ACCEPTED
     application.save()
```

Comment:
To avoid having issues with receivers that can possibly break and raise issues either move the signal emission to the end of the function or use `send_robust` (I prefer the first option, it doesn't risk not saving the application, but propagates errors to make debugging easy).

```suggestion
    application.status = BetaTestApplication.ACCEPTED
    application.save()
    
    # Send signal after saving application to avoid potential issues in receivers
    betapplication_accepted.send(sender=None, application=application)
```

Reply:
Done